css/index.css

```diff
@@ -1,3 +1,439 @@
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
 
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+*, *:before, *:after {
+    box-sizing: inherit;
+  }
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+.container {
+  width: 880px;
+  /* TB LR */
+  margin: 0 auto;
+}
+
+/* container media queries */
+
+@media (max-width: 800px){
+   .container {
+       width: 800px;
+   }
+
+}
+
+@media (max-width: 500px){
+    .container {
+        width: 500px;
+    }
+ 
+ }
+
+/* header */
+
+nav {
+	display: flex;
+	justify-content: space-between;
+	width: 880px;
+    margin: 0 auto;
+
+
+}
+
+nav {
+	margin-top: 25px;
+}
+
+.nav-item {
+    display: flex;
+}
+
+.nav-item a {
+  margin-right: 65px;
+  font-size: 14px;
+  color: gray;
+  text-decoration: none;
+}
+
+
+
+.nav-item a:hover {
+  text-decoration: underline;
+}
+
+.nav-item a:first-child {
+  margin-left: 0;
+}
+
+.nav-item a:last-child {
+  margin-right: 0;
+}
+
+nav {
+	align-items: center;
+ 
+}
+
+/* nav bar media queries - tablet */
+
+@media (max-width: 800px) {
+    nav {
+        width: 800px;
+        flex-direction: column-reverse;
+    }
+
+    .nav-item {
+        justify-content: space-between;
+        width: 90%;
+        margin: 0 40px;
+    }
+
+    .nave-img {
+        margin: 25px 0 35px 0;
+    }
+
+}
+
+@media (max-width: 500px) {
+    nav {
+        width: 500px;
+        margin-top: 0;
+    }
+
+    .nave-img {
+        margin: 15px 0;
+    }
+
+    .nav-item {
+        width: 500px;
+        flex-direction: column;
+        align-items: center;
+    }
+
+    .nav-item a {
+        width: 100%;
+        margin: 0;
+        padding: 10px 0;
+        font-size: 150%;
+        border-top: 1px solid gray;
+        text-align: center;
+    }
+
+    .nav-item a:last-child {
+        border-bottom: 1px solid gray;
+    }
+}
+
+
+/* cta */
+.cta {
+	display: flex;
+}
+
+.cta .cta-text {
+  vertical-align: top;
+  font-size: 77px;
+  margin-top: 120px;
+  text-align: center;
+  padding: 0px 75px;
+}
+
+.cta .cta-text button {
+  background: white;
+  vertical-align: top;
+  margin-top: 6px;
+  padding: 8px 44px;
+  font-size: 17px;
+  border: 1px solid black;
+  cursor: pointer;
+}
+
+.cta .cta-text button:hover {
+  background: black;
+  color: white;
+}
+
+.cta img {
+  margin-top: 69px;
+}
+
+/* header content media queries*/
+
+@media (max-width: 800px) {
+    .cta {
+        justify-content: center;
+        margin-bottom: 60px;
+    }
+
+    .cta img {
+        display: none;
+    }
+
+    .cta .cta-text {
+        margin-top: 80px;
+    }
+}
+
+@media (max-width: 500px) {
```

Comment:
Try to not have multiple media queries that call the same arguments, it lowers the future manageability of the code